src/messages/accumulating.rs

```diff
@@ -31,18 +34,22 @@ pub struct AccumulatingMessage {
 impl AccumulatingMessage {
     /// Create new `AccumulatingMessage`
     pub fn new(
-        content: RoutingMessage,
+        src: Prefix<XorName>,
+        dst: DstLocation,
+        variant: Variant,
         section_share: &SectionKeyShare,
         public_key_set: bls::PublicKeySet,
         proof: SectionProofSlice,
     ) -> Result<Self> {
-        let bytes = serialize(&content)?;
+        let bytes = serialize(&(&src, &dst, &variant))?;
```

Comment:
Question: do we need the `src` in the thing that we signed, it is us after all and will be in the self validating SrcAuthority?

Reply:
Not sure. Say we take it out - is it guarantee that the src prefix is genuine then?

Reply:
Well, the src prefix + version will be the thing that we use to find the key on receiving a message, so if it is not right then it will not validate? (otherwise it is the top one, and it was signed by previous keys).